app/models/transaction.rb

```diff
@@ -2,28 +2,39 @@
-#  automatic_confirmation_after_days :integer
-#  community_id                      :integer          not null
-#  created_at                        :datetime         not null
-#  updated_at                        :datetime         not null
-#  starter_skipped_feedback          :boolean          default(FALSE)
-#  author_skipped_feedback           :boolean          default(FALSE)
-#  last_transition_at                :datetime
-#  current_state                     :string(255)
-#  commission_from_seller            :integer
-#  minimum_commission_cents          :integer          default(0)
-#  minimum_commission_currency       :string(255)
-#  payment_gateway                   :string(255)      default("none"), not null
-#  listing_quantity                  :integer          default(1)
-#  listing_author_id                 :string(255)
-#  listing_title                     :string(255)
-#  unit_price_cents                  :integer
-#  unit_price_currency               :string(8)
-#  payment_process                   :string(31)       default("none")
+#  id                                 :integer          not null, primary key
+#  starter_id                         :string(255)      not null
+#  listing_id                         :integer          not null
+#  conversation_id                    :integer
+#  automatic_confirmation_after_days  :integer
+#  community_id                       :integer          not null
+#  created_at                         :datetime         not null
+#  updated_at                         :datetime         not null
+#  starter_skipped_feedback           :boolean          default(FALSE)
+#  author_skipped_feedback            :boolean          default(FALSE)
+#  last_transition_at                 :datetime
+#  current_state                      :string(255)
+#  commission_from_seller             :integer
+#  minimum_commission_cents           :integer          default(0)
+#  minimum_commission_currency        :string(255)
+#  payment_gateway                    :string(255)      default("none"), not null
+#  listing_quantity                   :integer          default(1)
+#  listing_author_id                  :string(255)
+#  listing_title                      :string(255)
+#  unit_price_cents                   :integer
+#  unit_price_currency                :string(8)
+#  payment_process                    :string(31)       default("none")
+#  delivery_method                    :string(31)       default("none")
+#  shipping_price_cents               :integer
+#  shipping_address_status            :string(255)
+#  shipping_address_city              :string(255)
+#  shipping_address_country           :string(255)
+#  shipping_address_name              :string(255)
+#  shipping_address_phone             :string(255)
+#  shipping_address_postal_code       :string(255)
+#  shipping_address_state_or_province :string(255)
+#  shipping_address_street1           :string(255)
+#  shipping_address_street2           :string(255)
```

Comment:
I think we should pull these to a separate table. It would make sense both logically and in terms of storage / query efficiency. Majority of the transactions still won't have a shipping address attached to them.